OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -1,834 +1,1198 @@
+        const auto* frame = captureManager.GetSelectedFrame();
+
+        if (!frame)
+        {
+            OLO_CORE_ERROR("Cannot export CSV: no frame selected");
+            return false;
+        }
+
+        try
+        {
+            std::ofstream file(outputPath);
+            if (!file.is_open())
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)